tests/upgrade_v2_script/static/fluent_bit.output.yaml

```diff
@@ -11,6 +11,27 @@ fluent-bit:
       valueFrom:
         fieldRef:
           fieldPath: metadata.namespace
+  extraVolumes:
```

Comment:
Isnt't input and output the same? 🤔

Reply:
Let's pretend this didn't happen. PTAL now

Reply:
Actually to answer your question: it is. But that's ok right?

https://github.com/helm/charts/blob/7e45e678e39b88590fe877f159516f85f3fd3f38/stable/fluent-bit/values.yaml#L198-L207

vs 

https://github.com/fluent/helm-charts/blob/e6805d4509868b76717677c7f7eecc4834d49d5a/charts/fluent-bit/values.yaml#L111-L113

Reply:
👍